challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<p>`it might be worth adding a ignore warnings command at the start for producing the final output that will be stored on github for people to browse, just so that its a bit neater</p><p>(from https://stackoverflow.com/questions/14463277/how-to-disable-python-warnings )</p><pre class="ql-syntax" spellcheck="false">import warnings
warnings.filterwarnings('ignore')

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks. Annoyingly this string of warnings popped up halfway through development